operations/governance-remediation-process.md

```diff
@@ -0,0 +1,83 @@
+
+**Public Reports: **In most cases, project governance issues will be reported publicly to the TOC by filing [an Issue in the TOC repository](https://github.com/cncf/toc/issues) of type Governance. Should a public issue be reported in some other venue (the project's issue tracker, mailing list, etc.) an issue in the TOC repo should be created for tracking and assignment.
+
+**Anonymous Reports**: Should the reporter not wish their name to be known, they should email &lt;TOC-confidential-email-here>.  The TOC team will then create an issue in the TOC repo tracking the remediation. The identity of the anonymous reporter will be known only to the TOC team.
+
+**Confidential Reports**: Should there be risks around the details of the remediation request being public, the reporter will email &lt;TOC-confidential-email-here>.  The TOC and the assessment team will then use privately shared documents to track the status of the remediation request.
+
+## Governance Remediation Process
+
+### Roles
+
+* TAG Contributor Strategy: For missing governance the assignee will lead, consulting and informing with the assigned TOC member as needed. For non-practiced governance, they act as an advisor to the TOC
+* TOC member assignee(s): serves an oversight/informational function that the governance assignee can leverage as needed to impress upon the project the needed changes or leads the discussion with the project in the case of non-practiced governance
+* Code of Conduct Committee: if a failure of governance coincides with CoC violations by one or more project leaders, the CoCC will coordinate with the assessment team.
+
+### Missing Governance
+
+1. Assessment Team Formation: The TOC will assign one TAG Contributor Strategy assignee and one TOC member.
+
+2. Investigation: The assessment team will Interview the reporter, project maintainers, and other relevant contributors to determine if the governance issue is valid and assess the project's ability to resolve it internally.
+
+3. Recommendation Development: In collaboration with the project, the assessment team will draft new governance policies to address gaps. This will include an actionable plan for the project to follow, with specific milestones and deadlines.
+
+4. Implementation and Monitoring: The assessment team will regularly check in on the project's progress towards defined milestones, provide additional support as needed, and provide a public progress report at 3 and 6 month milestones.
+
+5. Resolution or Escalation: If the project meets the milestones, the issue can be resolved. If not, it should be transferred to the Non-Practiced Governance process.
+
+### Governance Remediation Process: Non-Practiced Governance
+
+1. Triage: The TOC will do an initial triage of the issue and assign a lead member and a supporting member to guide the process.
+
+2. Assessment Team Formation: The two TOC members will work with a member of WG Governance to form an assessment team.
```

Comment:
Line 33 talks about "Governance WG" and this line talks about "WG Governance"; neither are ever defined. The rest of the document talks about TAG Contributor Strategy.

Reply:
Yeah, we should add links to the GovWG here.

Reply:
Looks like the link to the WG was added. Resolving this thread.